ci-operator/step-registry/telco5g/ptp/tests/telco5g-ptp-tests-commands.sh

```
@@ -5,216 +5,25 @@ set -o nounset
-  # print the build logs
-  oc -n openshift-ptp logs podman
-
-  if [[ $success -eq 1 ]]; then
-    echo "[INFO] index build succeeded"
+get_ci_images() {
+  if [[ "${T5CI_DEPLOY_UPSTREAM:-false}" == "true" ]]; then
+    export IMG="${PTP_OPERATOR_IMAGE:?PTP_OPERATOR_IMAGE not set by ci-operator dependency}"
+    export DAEMON_IMG="${PTP_DAEMON_IMAGE:?PTP_DAEMON_IMAGE not set by ci-operator dependency}"
+    local release_image
+    release_image=$(oc get clusterversion version -o jsonpath='{.status.desired.image}')
+    export SIDECAR_IMG=$(oc adm release info "${release_image}" --image-for=cloud-event-proxy)
```

Shellcheck (0.11.0)

[warning] Declare and assign separately to avoid masking return values.

(SC2155)